server/cmwell-cons/src/main/scala/Grid.scala

```diff
@@ -105,7 +106,9 @@ case class Grid(user: String,
         index = 1,
         rs = IpRackSelector(),
         g1 = g1,
-        hostIp = host
+        hostIp = host,
+        casDataDirs = IndexedSeq("cas"),
+        casDurableWrites = false
```

Comment:
please add comment: `// we refrain from using C* Commitlog on cluster, to save disk space and performance, given we always write in Quorum so there will be no data loss`

Reply:
done